server/plugin/api_test.go

```diff
@@ -1906,6 +1906,7 @@ func TestAPICreateIncident(t *testing.T) {
 		RequestBody          string
 		SetupAPI             func(*plugintest.API)
 		SetupClient          func(client *mock_plugin.Client)
+		SetupPlugin          func()
```

Comment:
```suggestion
		SetupPlugin          func(p *Plugin)
```

Reply:
we are also mocking some functions of different type. so I think it's okay to keep it like this